jira/resources.py

```diff
@@ -315,15 +307,15 @@ def update(
     ):
         """Update this resource on the server.
 
-        Keyword arguments are marshalled into a dict before being sent. If this
-        resource doesn't support ``PUT``, a :py:exc:`.JIRAError` will be raised; subclasses that specialize this method
-        will only raise errors in case of user error.
+        Keyword arguments are marshalled into a dict before being sent. If this resource doesn't support ``PUT``, a :py:exc:`.JIRAError`
+        will be raised; subclasses that specialize this method will only raise errors in case of user error.
 
         Args:
             fields (Optional[Dict[str, Any]]): Fields which should be updated for the object.
-            async_ (bool): If true the request will be added to the queue so it can be executed later using async_run()
+            async_ (bool): The request will be added to the queue, so it can be executed later using async_run()
```

Comment:
I would prefer all bool arguments to start with the word "True"

e.g.

```suggestion
            async_ (bool): True adds the request to async queue, so it can be executed later using async_run()
```

as I this ensures the meaning is crystal clear, prevents bad refactors

Reply:
makes sense in a way :D